src/geometamaker/geometamaker.py

```diff
@@ -515,6 +515,25 @@ def describe_collection(directory, depth=numpy.iinfo(numpy.int16).max,
     config = Config()
     resource = resource.replace(config.profile)
```

Comment:
Now that we're potentially inheriting values from an existing document, the timing of this `resource.replace(config.profile)` starts to matter. If we do it here, then it potentially gets overwritten by data from the existing doc. If we do it at the very end of the function, then the user profile always takes precedence. I'm not 100% sure what is the "right" choice, but in `describe` we're replacing the profile at the very end, so I think we should make both the `describe*` functions consistent.

Reply:
Ahh you're right. I was reading `resource = existing_metadata.replace(resource)` incorrectly and thinking it was doing the replacement in the other direction.